src/facades.php

```diff
@@ -101,7 +102,7 @@ function graphql(
     callable $fieldResolver = null
 ): ExecutionResult {
     /** @noinspection PhpParamsInspection */
-    return Execution::execute(
+    return executor()->execute(
```

Comment:
I would prefer `GraphQL::get(ExecutionInterface::class)->execute(` here.

Reply:
Technically I can call only `graphql(...)` but for some reason all the tests failed.